src/controllers/simplified-account/settings/webhooks/create/create.controller.js

```diff
@@ -10,10 +14,47 @@ async function get (req, res) {
   })
 }
 
-async function post (req, res) {
-  // TODO: implement post controller
-  res.redirect(formatSimplifiedAccountPathsFor(paths.simplifiedAccount.settings.webhooks.index, req.service.externalId, req.account.type)
-  )
+async function post (req, res, next) {
+  const accountIsLive = req.account.type === 'live'
+  const validations = [
+    webhookSchema.callbackUrl.validate,
+    webhookSchema.description.validate,
+    webhookSchema.subscriptions.validate
+  ]
+  await Promise.all(validations.map(validation => validation.run(req)))
+  const validationErrors = validationResult(req)
+  if (!validationErrors.isEmpty()) {
+    const formattedValidationErrors = formatValidationErrors(validationErrors)
+    return responseWithErrors(req, res, formattedValidationErrors)
+  }
+  try {
+    await webhooksService.createWebhook(req.service.externalId, req.account.id, accountIsLive, req.body)
+    res.redirect(formatSimplifiedAccountPathsFor(paths.simplifiedAccount.settings.webhooks.index, req.service.externalId, req.account.type))
+  } catch (createWebhookError) {
+    if (createWebhookError.errorIdentifier in webhookErrorIdentifiers) {
+      const callbackErrorMessage = webhookErrorIdentifiers[createWebhookError.errorIdentifier]
+      const callbackUrlError = {
+        errorSummary: [{ text: callbackErrorMessage, href: '#callback-url' }],
+        formErrors: { callbackUrl: callbackErrorMessage }
+      }
+      return responseWithErrors(req, res, callbackUrlError)
+    } else {
+      next(createWebhookError)
+    }
+  }
+}
+
+const responseWithErrors = (req, res, errors) => {
+  const subscriptions = typeof req.body.subscriptions === 'string' ? [req.body.subscriptions] : req.body.subscriptions
```

Comment:
Why would the type of subscriptions be different here? Feels like it should always be the same type

Reply:
if only one box is checked it goes into the req body as a string. If it's multiple then it's an array. I think this is just how that component works (is the same with card types)

Reply:
I see. That's unfortunate.